modules/cluster-logging-rn-5.5.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//cluster-logging-release-notes
+[id="cluster-logging-release-notes-5-5-0"]
+= Logging 5.5
+The following advisories are available for logging 5.5:
+link:https://access.redhat.com/errata/RHSA-2022:[Release 5.5]
+
+
+[id="openshift-logging-5-5-0-enhancements"]
+== Enhancements
+* With this update, you can forward structured logs from different containers within the same pod to different indices. To use this feature, you must configure the pipeline with multi-container support, and annotate the pods. (link:https://issues.redhat.com/browse/LOG-1296[LOG-1296])
+
+[IMPORTANT]
+====
+JSON formatting of logs varies by application. This feature should be used sparingly to create indices to separate logs with incompatible JSON formats, as creating too many indices will impact performance. To separate logs from different namespaces, or applications with compatible JSON formats, use queries.
```

Comment:
```suggestion
JSON formatting of logs varies by application. Because creating too many indices impacts performance, limit your use of this feature to creating indices for logs that have incompatible JSON formats. Use queries to separate logs from different namespaces, or applications with compatible JSON formats.
```